### PR TITLE
fix(api): move request logger to individual routes

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#1669

Ticket: #1669

### Dependencies

- Upstream: https://github.com/metriport/metriport-internal/pull/1685

### Description

Move the request logs to the individual routes in order to extract the params from the URL's

### Testing

_[Regular PRs:]_

- Local
  - [x] Test sending events to posthog
- Staging
  - [ ] Test sending events to posthog
- Production
  - [ ]Monitor

### Release Plan

- [ ] Upstream dependencies are met/released
- [ ] Merge this

### CODE REVIEW

File: packages/api/src/routes/medical/internal-patient.ts [hunk 1]
```diff
@@ -607,6 +624,7 @@ router.get(
  */
 router.post(
   "/trigger-update",
+  requestLogger,
```
Comment (RamilGaripov) [inline packages/api/src/routes/medical/internal-patient.ts:627]: I know this was not in the scope of this PR, but this route is of no use for us anymore - it was a one-time action back when we first added address coordinates. 

If you end up sending another commit and could delete it and any reference to it, that would be good

File: packages/api/src/routes/internal.ts [hunk 2]
```diff
@@ -210,6 +218,7 @@ router.get(
  */
 router.post(
   "/check-api-quota",
```
Comment (RamilGaripov) [inline packages/api/src/routes/internal.ts:220]: Not sure if we want this one to be logged. Do we?

File: packages/api/src/routes/medical/internal-cq.ts [hunk 3]
```diff
@@ -59,6 +60,7 @@ const sequelize = initDBPool(Config.getDBCreds());
  */
 router.post(
   "/directory/rebuild",
+  requestLogger,
```
Comment (RamilGaripov) [inline packages/api/src/routes/medical/internal-cq.ts:63]: This route is hit with the scheduled lambda once a day. Should we even log this?

File: packages/api/src/routes/medical/internal-cq.ts [hunk 4]
```diff
@@ -76,6 +78,7 @@ router.post(
 router.post(
   "/directory/insert",
   upload.single("file"),
+  requestLogger,
```
Comment (RamilGaripov) [inline packages/api/src/routes/medical/internal-cq.ts:81]: ^ ditto

**Review by RamilGaripov — APPROVED** [review]
Approved, but left some comments for consideration

File: packages/api/src/routes/medical/internal-patient.ts (hunk 1 above)
Comment (Orta21) [reply packages/api/src/routes/medical/internal-patient.ts:627]: Ill remove it the logger but leave the endpoint for now

**Pull request merged**